.circleci/config.yml

```diff
@@ -350,7 +350,7 @@ jobs:
       - run:
           name: staticcheck
           command: |
-            go install honnef.co/go/tools/cmd/staticcheck
+            go install honnef.co/go/tools/cmd/staticcheck@latest
```

Comment:
If this works I think we probably want an issue to track so that we can go back to a released version. I suspect that this is going to be brittle...

Reply:
I looked at this more closely and `@latest` pulls 0.5.1 so the upstream is handling things reasonably. Resolving this comment.